TRENZ.Docs.API/Interfaces/ISafeFileSystemPathTraversalService.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace TRENZ.Docs.API.Interfaces;
+
+public interface ISafeFileSystemPathTraversalService
+{
+    string Traverse(string root, string path);
```

Comment:
"Traverse" is not the correct verb for what the method does, I think. Maybe "SafeCombine"? Or "VerifyCombine"?

Reply:
Hmmmmmm. I don't love SafeCombine but sure.